Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings

```diff
@@ -227,7 +227,7 @@
 "ROUTE_QUALITY_ROADS_TOO_NARROW_FEEDBACK" = "Route had roads too narrow to pass";
 
 /* Label above the speed limit in an MUTCD-style speed limit sign. Keep as short as possible. */
-"SPEED_LIMIT_LEGEND" = "Max";
+"SPEED_LIMIT_LEGEND" = "Speed Limit";
```

Comment:
Is Android following similar approach? If so, do they localize this string to other languages as well? Can we re-use them?